ecosystem/lava-sdk/src/codec/lavanet/lava/common/fixationEntry.ts

```diff
@@ -21,6 +21,18 @@ export interface Entry {
   isLatest: boolean;
 }
 
+export interface GenesisEntries {
```

Comment:
why are there changes in the javascript proto files?

Reply:
because after you change proto files, you should regenerates the protobufs for the SDK. should I revert it?

Reply:
in the fixation?